### PR TITLE
Minor fixes to stores

### PR DESCRIPTION
**Pull request by Aljullu**
Small PR with some minor fixes to several stores.

### Detailed test instructions:
Fixes are very small and they shouldn't have any impact in the way wc-admin works, but an easy way to test would be going to every report page and making sure everything (summary numbers, chart and table data) still loads.

### CODE REVIEW

File: client/store/orders/selectors.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { ERROR } from 'store/constants';
  * Returns orders for a specific query.
  *
  * @param  {Object} state     Current state
- * @param  {Object} query     Report query paremters
+ * @param  {Object} query     Report query paremeters
```
Comment (allendav) [inline client/store/orders/selectors.js]: parameters

File: client/store/products/selectors.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import { getJsonString } from 'store/utils';
  * Returns products report details for a specific report query.
  *
  * @param  {Object} state  Current state
- * @param  {Object} query  Report query paremters
+ * @param  {Object} query  Report query paremeters
```
Comment (allendav) [inline client/store/products/selectors.js]: parameters

File: client/store/reports/stats/selectors.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ import { getJsonString } from 'store/utils';
  *
  * @param  {Object} state     Current state
  * @param  {String} endpoint  Stats endpoint
- * @param  {Object} query     Report query paremters
+ * @param  {Object} query     Report query paremeters
```
Comment (allendav) [inline client/store/reports/stats/selectors.js]: parameters

**Review by allendav — CHANGES_REQUESTED** [review]
A few small changes needed

File: client/store/orders/selectors.js (hunk 1 above)
Comment (Aljullu) [reply client/store/orders/selectors.js]: Ouch. :upside_down_face: 

Should be fixed everywhere, now. 614d905fa0921845362b4ad0987188e834253713

File: client/store/products/resolvers.js [hunk 4]
```diff
@@ -15,7 +20,7 @@ export default {
 
 		try {
 			const params = query ? '?' + stringify( query ) : '';
-			const products = await apiFetch( { path: '/wc/v3/reports/products' + params } );
+			const products = await apiFetch( { path: NAMESPACE + 'reports/products' + params } );
```
Comment (timmyc) [inline client/store/products/resolvers.js]: Nice trash pickup here!

**Review by timmyc — APPROVED** [review]
This is looking good to me and did not notice any regressions in testing. Thanks for cleaning these items up @Aljullu

**Pull request merged**